src/openforms/forms/utils.py

```diff
@@ -63,6 +68,12 @@ def form_to_json(form_id: int) -> dict:
 
     form_logic = FormLogic.objects.filter(form=form)
 
+    # Export only user defined variables and static variables
+    # The component variables should be regenerated from the form definition configuration
```

Comment:
This means that old versions without variables in them are forwards-compatible, right? Except for the static variables, so maybe we'll need to do a check during restoring if all static variables are present?

Reply:
Yes indeed. I was unsure if the static variables should be recreated for every restore, but maybe they should be like the component variables. We can have a 'list' of all static variables that forms should have and it can be used also in the data migration :thinking:

Reply:
that sounds like a solid approach!